jsx/Form.js

```diff
@@ -1229,8 +1297,8 @@ NumericElement.defaultProps = {
   name: '',
   min: null,
   max: null,
-  label: '',
-  value: '',
+  label: null,
+  value: null,
```

Comment:
not in **24.1**

_added to improve form functionality and UI_

to include as **CBIG override** with plans to push to **main** in some capacity